man/ostree.repo-config.xml

```diff
@@ -130,9 +130,10 @@ Boston, MA 02111-1307, USA.
      <varlistentry>
         <term><varname>min-free-space-size</varname></term>
         <listitem><para>Value (in MB, GB or TB) that specifies a minimum space in the
-        underlying filesystem to keep free. Also, note that min-free-space-percent
-        and min-free-space-size are mutually exclusive. Examples of acceptable values:
-        500MB, 1GB etc. The default value is 0MB, which disables this check.
+        underlying filesystem to keep free. Examples of acceptable values: 500MB, 1GB
+        etc. If this option is unset, min-free-space-percent is used instead.</para>
+        <para>If this option and min-free-space-percent are both set to non-zero
```

Comment:
nitpick: For consistency, I would use same wording as of `min-free-space-percent` more or less:

`In case this option is co-existing with min-free-space-percent (see above), this option takes priority.`